### DISCUSSION TITLE
Show institution admin users under each collection

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5892] [Feature Requests]
![image](https://github.com/specify/specify7/assets/37256050/de83c9d1-6418-4b28-9944-aa420594aa11)

Institution admins show up at the Institution level, but not under all of the collections, despite having access to all of them.

They _do_ show up if you manually grant them permission to a collection _before_ making them an institution admin.

Making this behavior consistent would be very nice.

> The case came up in that the mcruz account created the database and was the first user, so it is listed both at the institution level and under the collection.
> kspeer was created from the institution level page so the account did not appear under the collection.
> I went back and recreated the account at the collection level so I can see it there at at the institution level.
> 
> This seems buggy [or an incomplete feature]. As a collection admin, I want to be able to view the accounts that have access to the collection and see the institution admins listed among them. Not have to jump around to see who sees what.

**Reported By:** Matthew Cruz at UMich

**Comment by maxpatiiuk** [comment]
this bug might have resulted from miscommunication between me and Ben

there is a back-end endpoint that returns all the users that have any policies in this collection - https://sp7demofish.specifycloud.org/permissions/user_policies/65536/ (where ID is collection id)

the "Institution Admin" policy is not scoped to any collection

when developing front-end, I probably assumed that this endpoint would return that policy too.

---

solution:

Institution page calls this hook to get admins - https://github.com/specify/specify7/blob/xml-editor/specifyweb/frontend/js_src/lib/components/Security/Institution.tsx#L54

instead, move that hook to be in SecurityOutlet so that you can fetch it just once and use both in Institution and Collection pages. then modify the list of users on the collection page to also include institution admins if they aren't there already
